src/Magnum/ImGuiIntegration/Context.cpp

```diff
@@ -9,7 +9,7 @@
     Copyright © 2018 Jonathan Hale <squareys@googlemail.com>
     Copyright © 2019 bowling-allie <allie.smith.epic@gmail.com>
     Copyright © 2021 Juan Pedro Bolívar Puente <raskolnikov@gnu.org>
-    Copyright © 2022, 2024 Pablo Escobar <mail@rvrs.in>
+    Copyright © 2022, 2025 Pablo Escobar <mail@rvrs.in>
```

Comment:
```suggestion
    Copyright © 2022, 2024, 2025 Pablo Escobar <mail@rvrs.in>
```

Same in the other cases.